deploy/sre-prometheus/100-multiple-efs-operator-PrometheusRule.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: aws-efs-alerts
+    role: alert-rules
+  name: aws-efs-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: aws-efs-alerts 
+    rules:
+    - alert: MultipleVersionsOfEFSCSIDriverInstalled 
+      expr: count(kube_pod_info{namespace="openshift-operators", pod=~"efs-csi-node-.*"}) > 0 and count(kube_pod_info{namespace="openshift-cluster-csi-drivers", pod=~"aws-efs-csi-driver-operator-.*"}) > 0
+      labels:
+        severity: critical
+        namespace: openshift-monitoring
+        source: "https://issues.redhat.com/browse/OSD-16223"
+        link: "https://github.com/openshift/ops-sop/blob/master/v4/alerts/MultipleVersionsOfEFSCSIDriverInstalled.md"
```

Comment:
the link throws 404 for me. For a better primary experience, it might be nice to implement this beforehand

Reply:
FYI, I think this should be populated automatically https://github.com/openshift/configure-alertmanager-operator/blob/master/controllers/secret_controller.go#L504